backend/app/routes/ownerRoutes.js

```diff
@@ -1,15 +1,15 @@
-const express = require("express");
-const { updateOrder, getActiveOrders, getTables, getTable, closeTable, clearCallWaiter } = require("../controllers/ownerController");
+const express = require('express');
+const { updateOrderController, getActiveOrdersController, getTableByRestaurantIdController, getTableByIdController, closeTableController, callWaiterController } = require('../controllers/ownerController');
 
 const router = express.Router();
 
-router.post("/orders/update/:orderId", updateOrder);
-router.get("/orders/active/:restaurantId", getActiveOrders);
+router.post('/orders/update/:orderId', updateOrderController);
+router.get('/orders/active/:restaurantId', getActiveOrdersController);
 
-router.get("/tables/:restaurantId", getTables);
+router.get('/tables/:restaurantId', getTableByRestaurantIdController);
 
-router.get("/table/:tableId", getTable);
-router.get("/table/close/:tableId", closeTable);
+router.get('/table/:tableId', getTableByIdController);
+router.get('/table/close/:tableId', closeTableController);
 
-router.get("/actions/clearCallWaiter/:tabId", clearCallWaiter);
+router.get('/actions/clearCallWaiter/:tabId', callWaiterController);
 module.exports = router;
```

Comment:
This code patch replaces the existing route functions with new controllers in the `ownerController`. The changes seem to be mostly cosmetic replacing single-quotes with double-quotes and renaming function names to use "Controller" suffix. 

From this code patch, it is not possible to determine if there are any bugs or risks that could cause issues at runtime. However, it would be useful to inspect the new functions in `ownerController` for potential coding mistakes. Overall, it seems like a reasonable update to improve consistency and readability.